script.js

```diff
@@ -167,7 +190,7 @@
 
         if ( requestPage && requestPage.length ) {
             var $entry = jQuery('<li/>').addClass('requestPage').text(requestPage).appendTo($appendTo);
-            guiElementActions(actions, requestPage, $entry);
+            guiElementActions([ORPHANACTIONS.view('Page')], requestPage, $entry);
```

Comment:
Nope, this should only affect the requestPages. I.e. the pages listed below wanted/linked pages/media. These also only have a view action.

Or did I miss a use case of the plugin?